server/controllers/inventory/index.js

```diff
@@ -88,6 +95,76 @@ function inventoryLog(req, res, next) {
   }).catch(next);
 }
 
+// get inventory log as excel
+// GET /inventory/download/log/:uuid?rendere=xlsx?lang=fr
```

Comment:
This URL doesn't match what the actual URL is.  Can you change the parameters to reflect that?